src/main/java/com/google/songgame/servlets/GameServlet.java

```diff
@@ -46,21 +53,129 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
     datastore.put(roundEntity);
   }
-  
 
-  private String getAuthToken() {
-    String clientId = "1c29ff191b444611a6d9dbb4a354642f";
-    String clientSecret = "6b92b6eb8fe24d5e98178407d3099bd8";
-    SpotifyApi spotifyApi = new SpotifyApi.Builder().setClientId(clientId).setClientSecret(clientSecret).build();
-    ClientCredentialsRequest clientCredentialsRequest = spotifyApi.clientCredentials().build();
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String playlistUrl = getParameter(request, "playlist-link", "");
+    setvideoId(playlistUrl);
+    response.sendRedirect("/game.html");
+  }
 
+  /**
+   * @return the request parameter, or the default value if the parameter was not specified by the
+   *     client
+   */
+  private String getParameter(HttpServletRequest request, String name, String defaultValue) {
+    String value = request.getParameter(name);
+    if (value == null) {
+      return defaultValue;
+    }
+    return value;
+  }
+
+  /**
+   * Use playlist url to connect to YouTube API and set video ID.
+   *
+   * @param playlistUrl
```

Comment:
Javadoc looks great. Note that you can remove the @param part if you aren't providing any additional information about the parameter. That same goes for other javadocs you added.